downstream/assemblies/platform/assembly-controller-search.adoc

```diff
@@ -0,0 +1,15 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-controller-search"]
+= {ControllerNameStart} search
+
+:context: search-controller
+
+{ControllerNameStart} has a powerful search tool providing both search and filter capabilities across multiple functions. Search criteria are provided in an expandable "cheat-sheet" accessible from the *Advanced* option from the *Name* menu in the search field. 
```

Comment:
Possible re-write, ignore if you wish:
"Use {ControllerName}'s search tool for search and filter capabilities across multiple functions. An expandable "cheat -sheet" with search criteria is available in the *Advanced* option from the *Name* menu of the search field."